packages/react-component-playground/src/components/welcome-screen.less

```diff
@@ -0,0 +1,20 @@
+.welcome-screen {
+  position: absolute;
+  left: 250px;
+  min-height: 100vh;
+  width: ~"calc(100% - 250px)";
+  display: flex;
+  justify-content: center;
```

Comment:
Just to conclude on the scroll-bar, this is the reason (Mac settings):

![screenshot 2017-01-29 10 44 46](https://cloud.githubusercontent.com/assets/462561/22402903/06629ae6-e610-11e6-816f-a175cec7b0dc.png)

Switching it to default fixes it 😃